### PR TITLE
[oneDNN] Fixing a failure in mkl_fused_batch_norm_op_test

### PR DESCRIPTION
**Pull request by mahmoud-abuzaina**
This PR fixes a failure in a test that was enabled recently by this [PR](https://github.com/tensorflow/tensorflow/commit/155b34bad741124f778978f806d82551f9d35a66). The reference values are slightly changed due to use of different gemm-api. That resulted in a one value mismatch. So we are updating the test by adding relative error tolerance.

### CODE REVIEW

File: tensorflow/core/kernels/mkl/mkl_fused_batch_norm_op_test.cc [hunk 1]
```diff
@@ -159,7 +159,7 @@ class CommonTestUtilities : public OpsTestBase {
     ASSERT_EQ(offset_backprop.shape(), mkl_offset_backprop.shape());
 
     test::ExpectClose(output, mkl_output, 1e-5);
-    test::ExpectClose(scale_backprop, mkl_scale_backprop, 1e-5);
+    test::ExpectClose(scale_backprop, mkl_scale_backprop, 1e-5, /*rtol*/ 1e-5);
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl/mkl_fused_batch_norm_op_test.cc:162]: Nit: For completeness.
```suggestion
    test::ExpectClose(scale_backprop, mkl_scale_backprop, /*atol=*/1e-5, /*rtol=*/1e-5);
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the fix!

File: tensorflow/core/kernels/mkl/mkl_fused_batch_norm_op_test.cc (hunk 1 above)
Comment (mahmoud-abuzaina) [reply tensorflow/core/kernels/mkl/mkl_fused_batch_norm_op_test.cc]: I have addressed the comment. Thank you!

**Review by penpornk — APPROVED** [review]
Thank you for the quick fix!

**Pull request merged**